src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3756,6 +3756,17 @@ void M2(string S1 = $""Testing"", object O = null, Namae N = null)
                     //                     goto case $"Number 3";
                     Diagnostic(ErrorCode.ERR_FeatureInPreview, @"$""Number 3""").WithArguments("constant interpolated strings").WithLocation(49, 31));
         }
+
+        [Fact]
+        public void EmptyConstInterpolatedString()
+        {
+            CreateCompilation(@"
+class C
+{
+    const string s = $"""";
```

Comment:
>const string s = $""""; [](start = 4, length = 23)

It feels like we should run and verify the value at runtime.

Reply:
Or verify the constant value after importing from metadata.

---
In reply to: [533796984](https://github.com/dotnet/roslyn/pull/49707#discussion_r533796984) [](ancestors = 533796984)